src/app/shared/services/common-dialog.service.ts

```diff
@@ -5,14 +5,18 @@ import { TransactionEvent } from '../models/transaction-event';
 import { Observable } from 'rxjs';
 import { TransactionEventType } from '../enums/transaction-event-type';
 import { DomSanitizer } from '@angular/platform-browser';
+import { KeysPasswordDialogComponent } from '../../key-store/keys-password-dialog/keys-password-dialog.component';
```

Comment:
There's no need to create dependencies between modules, just create new service inside `KeyStore` module i.e: `KeyStoreDialogService`